src/screens/OrganizationFundCampaign/modal/CampaignModal.tsx

```
@@ -123,11 +112,10 @@ const CampaignModal: React.FC<InterfaceCampaignModal> = ({
       await createCampaign({
         variables: {
           name: formState.campaignName,
-          currency: formState.campaignCurrency,
-          fundingGoal: formState.campaignGoal,
-          organizationId: orgId,
-          startDate: dayjs(formState.campaignStartDate).format('YYYY-MM-DD'),
-          endDate: dayjs(formState.campaignEndDate).format('YYYY-MM-DD'),
+          currencyCode: formState.campaignCurrency,
+          goalAmount: parseInt(formState.campaignGoal.toString()),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)